src/Couchbase.Lite.Tests.Shared/TLSIdentityTest.cs

```diff
@@ -116,7 +116,13 @@ public void TestImportIdentity()
         {
             TLSIdentity id;
             byte[] data = null;
-            using(var stream = typeof(TLSIdentityTest).GetTypeInfo().Assembly.GetManifestResourceStream("certs.p12"))
+            #if NET6_0_ANDROID
+            //Note: Maui Android cert requirement: https://stackoverflow.com/questions/70100597/read-x509-certificate-in-android-net-6-0-application
+            //When export the cert, encryption has to be TripleDES-SHA1, AES256-SHA256 will not work...
+            using (var stream = typeof(TLSIdentityTest).GetTypeInfo().Assembly.GetManifestResourceStream("certs.pfx"))
+            #else 
+            using (var stream = typeof(TLSIdentityTest).GetTypeInfo().Assembly.GetManifestResourceStream("certs.p12")) 
```

Comment:
I don't think there is a need to separate the certs.  The new cert should work everywhere